game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
There is already an event for this from the net table when the value changes. It's basically the exact same implementation on the JS side with less code on the lua side. It also keeps the management of score and score cap scoped to exactly 1 object who's modifications are tracked by the dota engine.

chrono is also right about moving that code, i'm pretty sure it's needed to make it work right. also that's really where it should be, since that's the chunk of code with authority over the `limit`.

points manager should still initialize the value for `score`, just not `limit`

also lua doesn't have a switch statement, you have to use a table to do something like that. it's really dumb.

hierarchy?? 😂